exporter/logzioexporter/exporter.go

```diff
@@ -16,24 +15,25 @@ import (
 	"time"
 
 	"github.com/hashicorp/go-hclog"
-	"github.com/jaegertracing/jaeger-idl/model/v1"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/consumer/consumererror"
 	"go.opentelemetry.io/collector/exporter"
 	"go.opentelemetry.io/collector/exporter/exporterhelper"
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/plog"
+	"go.opentelemetry.io/collector/pdata/plog/plogotlp"
 	"go.opentelemetry.io/collector/pdata/ptrace"
+	"go.opentelemetry.io/collector/pdata/ptrace/ptraceotlp"
 	"google.golang.org/genproto/googleapis/rpc/status"
 	"google.golang.org/protobuf/proto"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/logzioexporter/internal/cache"
```

Comment:
I dont think we use jaeger `cache` anymore, you can remove it from the imports, and delete the files

Reply:
also remove it from `logzioExporter`
```
type logzioExporter struct {
	config       *Config
	client       *http.Client
	logger       hclog.Logger
	settings     component.TelemetrySettings
	serviceCache cache.Cache
}
```